src/lib/salad/rope.h

```diff
@@ -704,7 +704,7 @@ rope_node_print(const struct rope_node *node, rope_visit_f print, void *cb_arg,
 	const char *padding[] = { "│   ", "   " };
 
 	rope_size_t child_prefix_len = strlen(prefix) + strlen(padding[0]) + 1;
-	char *child_prefix = malloc(child_prefix_len);
+	char *child_prefix = (char *)malloc(child_prefix_len);
```

Comment:
Why not use `xmalloc` as checkpatch suggests? I don't see why we can't include `trivia/util.h` into this file. We do include it into `salad/mhash.h` and `salad/stailq.h`.

Reply:
Done.